pyface/fields/i_combo_field.py

```diff
@@ -0,0 +1,90 @@
+#------------------------------------------------------------------------------
+# Copyright (c) 2019, Enthought, Inc.
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in enthought/LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+# Thanks for using Enthought open source!
+#
+# Author: Enthought, Inc.
+# Description: <Enthought pyface package component>
+#------------------------------------------------------------------------------
+""" The text field interface. """
+
+from six import text_type
```

Comment:
Using `six`... so, how do you feel about importing `__future__.[absolute_import|division|print_function|unicode_literals]`?

Reply:
Done